pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -421,6 +421,23 @@ func doActivate(ctx *zedmanagerContext, uuidStr string,
 	log.Functionf("doActivate for %s", uuidStr)
 	changed := false
 
+	// Do we try to activate an application earlier than it's configured to start?
```

Comment:
This code block should be below the network block, particularly if we want to change the state since that block might set it it AWAITNETWORKINSTANCE.

Reply:
So, we do not take into account the comments from here?
https://github.com/lf-edge/eve/pull/2652#discussion_r892206400
I just point to it if you missed it)